auth/auth.go

```diff
@@ -101,6 +108,23 @@ func (a ACL) VerifyPeerCertificateServer(rawCerts [][]byte, verifiedChains [][]*
 		return nil
 	}
 
+	// Check against OPA
+	if a.AllowOPAQuery != nil {
+		ctx, cancel := context.WithTimeout(context.Background(), time.Second)
```

Comment:
We can address this separately, but I think the timeout should probably be configurable (either use the connect timeout or provide a separate flag for it). But let's stick with 1s for now.

Reply:
Adding a note about that in the list of caveats for now